website/content/en/docs/kubebuilder-migration/_index.md

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
We can remove this index file since we don't want this migration doc to show up on the website without the integration done first.